src/helpers.ts

```diff
@@ -66,22 +67,24 @@ function printDoc(gridResult: [], columns: ColumnModel[], gridName: string): voi
     documentToPrint.document.close();
 }
 
-function exportFile(gridResult: [], columns: ColumnModel[]): void {
+function exportFile(gridResult: [], columns: ColumnModel[], gridName: string): void {
```

Comment:
I really like this. Just one thing: can you put a default value to it? gridName = "data". This is to avoid breaking changes.